aten/src/ATen/native/cuda/ReflectionPad.cu

```diff
@@ -397,17 +397,9 @@ void reflection_pad2d_backward_out_template(
 
 } // namespace
 
-
-Tensor& reflection_pad1d_out_cuda(const Tensor& input, IntArrayRef padding,
-    Tensor& output) {
+TORCH_IMPL_FUNC(reflection_pad1d_out_cuda)
+(const Tensor& input, IntArrayRef padding, const Tensor& output) {
   reflection_pad1d_out_template(output, input, padding);
```

Comment:
This will "work" but it will be better to inline reflection_pad1d_out_template  into the IMPL_FUNC and then delete the unnecessary shape checking code.